.forge-snapshots/update dynamic fee in before swap.snap

```diff
@@ -1 +1 @@
-157503
\ No newline at end of file
+124589
```

Comment:
please can you run the tests with `forge test --isolate` and push the snapshots? Looks to me like you ran them with `forge test`